pkgs/development/libraries/science/chemistry/openmm/default.nix

```diff
@@ -6,10 +6,16 @@
 , fftwSinglePrec
 , doxygen
 , swig
-, python3Packages, enablePython ? false
-, opencl-headers, ocl-icd, enableOpencl ? false
-, clang, enableClang ? true
-, cudatoolkit, enableCuda ? false
+, python3Packages
+, enablePython ? false
+, opencl-headers
+, ocl-icd
+, enableOpencl ? false
```

Comment:
I have not tried OpenCL. Does this also try to run test against CL hardware?

Reply:
very valid point. Just tried and it does and also fails to initialise. Will disable tests also for OpenCL.